### DISCUSSION TITLE
Manually Inputting RGB Values

### DISCUSSION THREAD

**Discussion by bryanscarolina** [SuperNova911/Gardener-for-GitHub#20] [Bug]
**Describe the bug**
Try entering any value in one of the  RGB fields and it will output another number. 237 becomes 127.

**To Reproduce**
- Create a new theme
 - Click the color dot to pull up the color picker
- click on the tab with the HEX number at the bottom and 3 RGB fields (Don’t remember if it’s Spectrum or what and not sure I can save this)
- change any of the RGB fields. HEX seems to be fine.

**Expected behavior**
Entering 250 250 250 outputs a shade of white with those numbers in place. I did check my keyboard in 2 color palette apps - ColorSlurp and Coolors. Seemed to be working normally.

**User information**
- Device: iPhone14,3
- OS: iOS 15.4
- App Version: 1.0.0 (21)

**Additional context**
Not a big deal. I just copied and pasted the number I wanted to get around it. I just wanted to let you know so some idiot doesn’t leave a 1-star review because of it .

**Comment by SuperNova911** [comment]
Thanks for report! I'll look into it 👀
